src/Hooks.ts

```diff
@@ -6,7 +6,7 @@ import type { Token, TokensList } from './Tokens.ts';
 
 export class _Hooks {
   options: MarkedOptions;
-  block: boolean | undefined;
+  block?: boolean;
```

Comment:
Note that these do not have the same meaning.

https://x.com/DavidKPiano/status/1856701854102557026

Although I think in this case that's okay.